src/services/maci/types.ts

```diff
@@ -89,3 +90,29 @@ export interface IDeployVkRegistryArgs {
    */
   tallyVotesZkeyPathNonQv: ISnarkJSVerificationKey;
 }
+
+export interface IRegisterArgs {
+  /**
+   * Id of the contract
+   */
+  id: EContracts;
+
+  /**
+   * Contract instance
+   */
+  contract?: BaseContract;
+
+  /**
+   * Contract address
+   */
+  address?: string;
+
+  /**
+   * args for ContractStorage
+   */
+  args?: unknown[];
+}
+
+export type IAbi = {
+  [key in EContracts]: Interface | InterfaceAbi;
+};
```

Comment:
```suggestion
export type IAbi = Record<EContracts, Interface | InterfaceAbi>;
```

Reply:
I think they are the same? or because this is easier to read?